### PR TITLE
GJBaseGameLayer stuff

### PR DESCRIPTION
**Pull request by TheSillyDoggo**

### CODE REVIEW

File: bindings/2.204/GeometryDash.bro [hunk 1]
```diff
@@ -6843,7 +6843,7 @@ class GJGameState {
 	TodoReturn controlTweenAction(int, int, GJActionCommand);
 	TodoReturn getGameObjectPhysics(GameObject*);
 	TodoReturn processStateTriggers();
-	TodoReturn stopTweenAction(int);
+	stopTweenAction(int) = win 0x18EA30;
```
Comment (dankmeme01) [inline bindings/2.204/GeometryDash.bro:6846]: wheres did the return go

Comment (TheSillyDoggo) [reply bindings/2.204/GeometryDash.bro:6846]: look it's 10pm

Comment (TheSillyDoggo) [reply bindings/2.204/GeometryDash.bro:6846]: actually can't figure out how to edit the code on mobile so uhhh

Comment (TheSillyDoggo) [reply bindings/2.204/GeometryDash.bro]: Fixed it

**Pull request merged**